doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -480,9 +452,8 @@ automatically created.
       scrutinee’s head, recursively, casts being ignored. :cmd:`Hint Mode` is
```

Comment:
The sentence on multiple modes above should be split out in its own note box. It's currently not visible enough (cf.  #13314).